Code/GraphMol/QueryOps.h

```diff
@@ -79,7 +79,18 @@ static inline int queryAtomTotalDegree(Atom const *at) {
   return at->getTotalDegree();
 };
 static inline int queryAtomHeavyAtomDegree(Atom const *at) {
-  return at->getTotalDegree() - at->getTotalNumHs(true);
+  int heavyDegree = 0;
+  ROMol::ADJ_ITER nbrIdx, endNbrs;
+  boost::tie(nbrIdx, endNbrs) = at->getOwningMol().getAtomNeighbors(at);
```

Comment:
not connected to this PR, but we should add range iterators for Atom neighbors and Atom bonds